src/Text/Parsing/Parser/Token.purs

```diff
@@ -1,14 +1,35 @@
 module Text.Parsing.Parser.Token where
 
 import Data.String
+import Data.Either
 
-import Control.Monad.State.Class
+import Control.Monad.State.Class hiding (get)
 import Control.Monad.Error.Class
 
 import Text.Parsing.Parser
 import Text.Parsing.Parser.String
 import Text.Parsing.Parser.Combinators
 
+takeTok :: forall m a. (Monad m) => ParserT [a] m a
```

Comment:
Can we call this something like `head` or `token`? I generally don't like to abbreviate, especially when the word is so short to begin with :)